src/structurizr/view/view.py

```diff
@@ -182,14 +195,21 @@ def _add_relationship(self, relationship: Relationship) -> RelationshipView:
             view = next(
                 (
                     rv
-                    for rv in self.relationship_views
+                    for rv in self._relationship_views
                     if rv.relationship is relationship
+                    and rv.description == description
+                    and rv.response == response
```

Comment:
Finding a relationship view seems to be a common operation, is it worth adding a method for, maybe?